src/MenuElementInterface.php

```diff
@@ -18,7 +18,7 @@
      *
      * @return string
      */
-    public function getValue();
+    public function getLabel();
```

Comment:
@mecha, absolutely agreeing with you. My only objection is that this method does not belong here ;)